src/mmg3d/API_functionsf_3d.c

```diff
@@ -137,6 +137,24 @@ FORTRAN_NAME(MMG3D_SET_OUTPUTSOLNAME,mmg3d_set_outputsolname,
   return;
 }
 
+/**
+ * See \ref MMG3D_Set_inputParamName function in \ref common/libmmgcommon_private.h file.
+ */
```

Comment:
I guess that the documentation issue comes from an error for another function but `MMG3D_Set_inputParamName` is declared in `mmg3d/libmmg3d.h` file.